### PR TITLE
Fix Unrecognized Amenities

### PR DESCRIPTION
**Pull request by lferreira**
So after 2 days we figured out this **f%$#ing** problem.

So we use to have lots of Amenities Mapped but one day we decided
to remove them. There are some Listings using those oldest Amenities
and in some cases we just forgot to declare in our map.

So this bug causes lots of URLNotifications with Amenity_NONE for URL's
like '.../com-playground/' '.../com-gramado/'

Ronaldo uses this information to generate sitemap and in this case to
skip those kind of Amenity but because we had Amenity_NONE we've been
generating wrong RP's sitemap

**NOW WE CAN SLEEP AGAIN**

### CODE REVIEW

File: amenities.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ var AmenitiesMetaFilter map[listingsEnums.Amenity]*MetaFilter = map[listingsEnum
 	listingsEnums.Amenity_ADULT_POOL:                &MetaFilter{fmt.Sprintf("Com %s", Slugify(Amenities[listingsEnums.Amenity_ADULT_POOL])), true},
 	listingsEnums.Amenity_AIR_CONDITIONING:          &MetaFilter{fmt.Sprintf("Com %s", Slugify(Amenities[listingsEnums.Amenity_AIR_CONDITIONING])), true},
 	listingsEnums.Amenity_ALARM_SYSTEM:              &MetaFilter{fmt.Sprintf("Com %s", Slugify(Amenities[listingsEnums.Amenity_ALARM_SYSTEM])), true},
-	listingsEnums.Amenity_Amenity_NONE:              &MetaFilter{fmt.Sprintf("Com %s", Slugify(Amenities[listingsEnums.Amenity_Amenity_NONE])), false},
```
Comment (brennovich) [inline amenities.go]: Why did we slugify nothing? hahah

File: amenities.go [hunk 2]
```diff
@@ -141,7 +141,7 @@ var Amenities map[listingsEnums.Amenity]string = map[listingsEnums.Amenity]strin
 	listingsEnums.Amenity_PLAYGROUND_2:              "Playground",
 	listingsEnums.Amenity_POOL:                      "Piscina",
 	listingsEnums.Amenity_RECEPTION:                 "Recepção",
-	listingsEnums.Amenity_REFLECTING_POOL:           "Espelhos d'água",
+	listingsEnums.Amenity_REFLECTING_POOL:           "Piscina",
```
Comment (brennovich) [inline amenities.go]: What a feature! REFLECTIONG POOL geezzzz

sic

http://g1.globo.com/distrito-federal/noticia/2016/10/carro-cai-em-espelho-dagua-do-palacio-do-itamaraty-em-brasilia.html

File: amenities.go [hunk 3]
```diff
@@ -157,45 +157,65 @@ var Amenities map[listingsEnums.Amenity]string = map[listingsEnums.Amenity]strin
...
-	"com-interfone":             listingsEnums.Amenity_INTERCOM,
-	"com-seguranca-24h":         listingsEnums.Amenity_SECURITY_24_HOURS,
-	"com-sistema-de-alarme":     listingsEnums.Amenity_ALARM_SYSTEM,
-	"com-vigia":                 listingsEnums.Amenity_WATCHMAN,
-	"com-area-de-servico":       listingsEnums.Amenity_SERVICE_AREA,
-	"com-cozinha":               listingsEnums.Amenity_KITCHEN,
-	"com-escritorio":            listingsEnums.Amenity_HOME_OFFICE,
-	"com-sacada":                listingsEnums.Amenity_BALCONY,
-	"com-varanda-gourmet":       listingsEnums.Amenity_GOURMET_BALCONY,
+	"com-salao-de-jogos":    listingsEnums.Amenity_ADULT_GAME_ROOM,
```
Comment (brennovich) [inline amenities.go]: ¬¬

Comment (brennovich) [reply amenities.go]: Wouldn't be better to split these legacy ones and add a proper documentation about keeping them?

File: amenities_test.go [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package zagallo
+
+import (
+	"testing"
+)
+
+func TestAmenitiesToSlugMapper(t *testing.T) {
```
Comment (brennovich) [inline amenities_test.go]: Would be great to test all [listings amenities](https://github.com/GrupoZapVivaReal/proto-resources/blob/master/vivareal/protobuf/listings/enums/Amenity.proto) and check them against our mappers, wouldn't it?

**Comment by lferreira** [comment]
Done @brennovich !

**Pull request merged**